src/cascadia/QueryExtension/AzureLLMProvider.cpp

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "AzureLLMProvider.h"
+#include "../../types/inc/utils.hpp"
+#include "LibraryResources.h"
+
+#include "AzureLLMProvider.g.cpp"
+#include "AzureResponse.g.cpp"
+
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Foundation::Collections;
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Xaml::Controls;
+using namespace winrt::Windows::System;
+namespace WWH = ::winrt::Windows::Web::Http;
+namespace WSS = ::winrt::Windows::Storage::Streams;
+namespace WDJ = ::winrt::Windows::Data::Json;
+
+static constexpr std::wstring_view acceptedModel{ L"gpt-35-turbo" };
```

Comment:
Since the code is being touched as part of this change: would you mind editing this to include the expanded list of models? I think this could be updated to a `std::array` of `std::wstring_view` entries and then further below, the query to verify that section of the response could search over that array.

Reply:
makes sense!